proposals/0295-codable-synthesis-for-enums-with-associated-values.md

```diff
@@ -227,13 +272,15 @@ Keys can be mapped to other names by conforming to `RawRepresentable`:
 ```swift
 enum Command: Codable {
   case load(key: String)
+  case store(key: String, Int)
 
   enum CodingKeys: String, CodingKey {
     case load = "lade"
   }
 
   enum LoadCodingKeys: String, CodingKey {
     case key = "schluessel"
+    case _1  = "wert"
   }
```

Comment:
Wouldn't the `case _1  = "wert"` cause a compiler diagnostic?
(The `.load` case only has a single associated value.)

Reply:
Ah, that should be `StoreCodingKeys` instead. Adding keys that don't match any parameters would cause a diagnostic, yes.